src/apiary.apib

```diff
@@ -1649,12 +1646,10 @@ level `conversions` array.
 + One Of
     + commission_rate: 4
     + commission_value
-    + commission_value_cpc
```

Comment:
Not as far as I can tell. When I try create a commission with it, the response says that 'commission_value_cpc' is not a valid field for the commission entity.